x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -4105,9 +4117,6 @@
         },
         "status": {
           "properties": {
-            "cancelled": {
```

Comment:
Removing these from the telemetry indices will require a re-index operation. Please make sure that you follow up with infra to let them know.